include/deal.II/lac/la_vector.h

```diff
@@ -214,7 +214,7 @@ namespace LinearAlgebra
      *  vec.locally_owned_elements() == complete_index_set(vec.size())
      * @endcode
      */
-    const dealii::IndexSet &locally_owned_elements() const;
```

Comment:
The main reason is that keeping an IndexSet is totally useless with Trilinos. All the information is in Epetra_Map. So I just want to create the IndexSet when it's necessary. This also matches our current interface.